lib/zen_ex/core/entities/ticket.ex

```diff
@@ -1,9 +1,35 @@
 defmodule ZenEx.Entity.Ticket do
-
-  defstruct [:id, :url, :external_id, :created_at, :updated_at, :type, :subject, :raw_subject,
-             :description, :priority, :status, :recipient, :requester_id, :submitter_id, :assignee_id,
-             :organization_id, :group_id, :collaborator_ids, :forum_topic_id, :problem_id, :has_incidents,
-             :due_at, :tags, :via, :ticket_form_id, :custom_fields, :satisfaction_rating, :sharing_agreement_ids]
+  defstruct [
```

Comment:
nitpick but the moduledoc should go first

Reply:
This how they've done it in their library but I'm up for making changes that reflect the community styleguide :+1:

Reply:
Baby steps. I'd probably work on the code first and then we can clean it up once we get some cred from the author 

![:parrotwave1:](https://cultofthepartyparrot.com/parrots/wave1parrot.gif) ![:parrotwave2:](https://cultofthepartyparrot.com/parrots/wave2parrot.gif) ![:parrotwave3:](https://cultofthepartyparrot.com/parrots/wave3parrot.gif) ![:parrotwave4:](https://cultofthepartyparrot.com/parrots/wave4parrot.gif) ![:parrotwave5:](https://cultofthepartyparrot.com/parrots/wave5parrot.gif) ![:parrotwave6:](https://cultofthepartyparrot.com/parrots/wave6parrot.gif) ![:parrotwave7:](https://cultofthepartyparrot.com/parrots/wave7parrot.gif) ![:parrotwave8:](https://cultofthepartyparrot.com/parrots/wave8parrot.gif) ![:parrotwave9:](https://cultofthepartyparrot.com/parrots/wave9parrot.gif)

Reply:
Yup! I'm trying to minimize the changes (many of these are the result of the formatter running) and focus on functionality for our needs. We can iterate on the overall code quality and test coverage.